cmd/clusterctl/log/logger_test.go

```diff
@@ -104,3 +105,40 @@ func TestFlatten(t *testing.T) {
 		})
 	}
 }
+
+func Test_logger_Enabled(t *testing.T) {
```

Comment:
nit: not sure if there is a naming convention on casing, but I'd call this `TestLoggerEnabled`

Reply:
This is just an autogenerated function name from the goland editor, let me rename it to the mentioned one to align with the existing code

Reply:
amended the changes with the fix, ptal